specifyweb/frontend/js_src/lib/components/TreeView/index.tsx

```diff
@@ -101,13 +102,19 @@ function TreeView<SCHEMA extends AnyTree>({
   // Node sort order
   const sortField = getPref(`${tableName as 'Geography'}.treeview_sort_field`);
 
+  const includeAuthor = getPref(`TaxonTreeEditor.DisplayAuthor`);
+
   const baseUrl = `/api/specify_tree/${tableName.toLowerCase()}/${
     treeDefinition.id
   }`;
 
   const getRows = React.useCallback(
     async (parentId: number | 'null') =>
-      fetchRows(`${baseUrl}/${parentId}/${sortField}`),
```

Comment:
Oh, you added `/` at the end of the URL!
This fixes a small bug (needles redirect) and thus will improve performance.

Reply:
Correct. In fact, if you find any other back-end API calls that don't have a trailing `/`, you should add it.